examples/time_series/goes_hek_m25.py

```diff
@@ -15,7 +15,7 @@
 
 ###############################################################################
 # Let's first grab GOES XRS data for a particular time of interest
-tr = TimeRange(['2011-06-07 04:00', '2011-06-07 12:00'])
+tr = TimeRange(['2011-06-07 00:00', '2011-06-07 12:00'])
```

Comment:
What caused this change?

Reply:
The goes `xrs` had one day files, so my implementation defaults the start time of a record to `00:00` on that day.
When the start time is given `4:00` it isn't less than `0:00` so scraper doesn't return that file.

Reply:
Can we revert this now?

Reply:
yes, will revert.